pkgs/by-name/mo/monkeysAudio/package.nix

```diff
@@ -22,10 +27,7 @@ stdenv.mkDerivation (finalAttrs: {
     description = "APE codec and decompressor";
     platforms = platforms.linux;
     mainProgram = "mac";
-    # This is not considered a GPL license, but it seems rather free although
-    # it's not standard, see a quote of it:
-    # https://github.com/NixOS/nixpkgs/pull/171682#issuecomment-1120260551
-    license = licenses.free;
+    license = licenses.bsd3;
```

Comment:
So they changed their license? Could you reference the upstream change?

Reply:
Was changed in version 10.18 see changelog for info https://monkeysaudio.com/versionhistory.html